my-ft-digest-promo/index.js

```diff
@@ -0,0 +1,86 @@
+const myFtClient = require('next-myft-client');
+const buttons = require('../buttons');
+const superstore = require('superstore-sync');
+const STORAGE_KEY = 'n-myft-digest-promo-seen';
+
+const CLASSES = {
+  ctaBtn: '.n-myft-digest-promo__cta-btn',
+  dismissBtn: '.n-myft-digest-promo__dismiss-btn',
+  promoComponent: '.n-myft-digest-promo',
+  promoEnabled: 'n-myft-digest-promo--enabled'
+};
+
+let btn, conceptId;
+
+function bindListeners() {
+  const btn = document.querySelector(CLASSES.ctaBtn);
+  const dismissBtn = document.querySelector(CLASSES.dismissBtn);
+
+  btn.addEventListener('click', addToDigest, false);
+  dismissBtn.addEventListener('click', () => {
+    hidePromo();
+    setDismissState();
+  }, false);
+}
+
+function shouldShowPromo(conceptId) {
+  return Promise.all([
+    myFtClient.get('followed', 'concept', conceptId),
+    myFtClient.get('preferred', 'preference', 'email-digest')
+  ]).then(values => {
+    return values[0].length === 0 && values[1].length === 0 && !getDismissState();
+  });
```

Comment:
This bit could be tidied up a bit by making use of [destructuring](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). e.g. 
```
.then([follows, prefers] => {
   return follows.length === 0 && prefers.length === 0 && !getDismissState();
})
```
(Also destructuring gets you mad hipster points)